sites/kit.svelte.dev/package.json

```
@@ -14,23 +14,24 @@
 		"@sveltejs/adapter-vercel": "workspace:^",
 		"@sveltejs/amp": "workspace:^",
 		"@sveltejs/kit": "workspace:^",
-		"@sveltejs/site-kit": "6.0.0-next.36",
-		"@types/d3-geo": "^3.0.3",
-		"@types/node": "^20.4.9",
+		"@sveltejs/site-kit": "6.0.0-next.42",
+		"@types/d3-geo": "^3.0.4",
+		"@types/node": "^20.6.0",
 		"browserslist": "^4.21.10",
 		"flexsearch": "^0.7.31",
-		"lightningcss": "^1.21.5",
-		"magic-string": "^0.30.2",
+		"lightningcss": "^1.21.8",
+		"magic-string": "^0.30.3",
 		"marked": "^9.0.0",
+		"prettier": "^3.0.3",
 		"prism-svelte": "^0.5.0",
 		"prismjs": "^1.29.0",
 		"shiki-twoslash": "^3.1.2",
-		"svelte": "^4.1.2",
+		"svelte": "^4.2.0",
 		"tiny-glob": "^0.2.9",
 		"typescript": "5.0.4",
 		"vite": "^4.4.9",
-		"vite-imagetools": "^5.0.7",
-		"vitest": "^0.34.1"
+		"vite-imagetools": "^5.0.8",
+		"vitest": "^0.34.4"
```

let's leave this one as is. I try to update the vitest version everywhere in the monorepo together at once to avoid pulling in multiple versions
```suggestion
		"vitest": "^0.34.1"
```